ChangeLog

```diff
@@ -1,6 +1,56 @@
 ======================================================================
 
 Originator: Jim Edwards
+Date: 01-30-2017
+Tag: cime5.2.0-alpha.23
+Answer Changes: None
+Tests: cesm_tests, scripts_regression_tests.py
+Dependencies:
+
+Brief Summary: new tfreeze_option setting for cesm, some system test refactoring for
```

Comment:
What PRs were in this tag? We should summarize this in every new ChangeLog summary.